.github/workflow_scripts/env_setup.sh

```diff
@@ -19,7 +19,8 @@ function setup_build_contrib_env {
 
 function setup_benchmark_env {
     # pip install -U autogluon.bench - revert back when latest update in PyPi
-    git clone https://github.com/autogluon/autogluon-bench.git
+    # git clone https://github.com/autogluon/autogluon-bench.git
+    git clone -b automm https://github.com/prateekdesai04/autogluon-bench.git
```

Comment:
It's not safe to clone from a master branch. It would be better to pin to a release tag.